stl/inc/algorithm

```diff
@@ -10102,9 +10102,14 @@ namespace ranges {
             if constexpr (forward_range<_Rng> && _Prefer_iterator_copies<iterator_t<_Rng>>) {
                 const auto _Found = _RANGES _Minmax_element_unchecked(
                     _STD move(_UFirst), _STD move(_ULast), _Pass_fn(_Pred), _Pass_fn(_Proj));
+                if (_Found.min == _Found.max) {

```

Comment:
For iterators that are just pointers, is the issue observable? 
If not, I'd check with `if constexpr` for that. Since unnecessary extra branch seems to be a significant penalty

Reply:
As far as I understand the issue is observable only with `move_iterator`-like iterators.
Ok, I will add `if constexpr` tomorrow.